projectOne/src/oneFour.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+public class oneFour {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int time [] = {341,273,278,329,445,402,388,275,243,334,412,393,299,343,317,265};
+        String names [] =  {"Elena","Thomas","Hamilton","Suzie","Phil","Matt","Alex","Emma","John","James","Jane","Emily","Daniel","Neda","Aaron","Kate"};
+        int temp;
+        String temp1;
+        for (int i = 1; i< time.length; i++) {
+            for (int j = i; j >0; j--) {
+                if (time[j] < time [j-1]) {
+                    temp = time[j];
+                    temp1 = names[j];
+                    time[j] = time[j-1];
+                    names[j] = names[j-1];
+                    time[j-1] = temp;
+                    names[j-1] = temp1;
+                }
+            }
+        }
```

Comment:
Bubble sorting isn't a better choice for this task. From my point of view, you should use a map or new class here